lib/components/toggles/index.styl

```
@@ -1,6 +1,7 @@
+@import '../../../stylesheets/common'
 @import '../../../stylesheets/defaults'
 
-:host
++host-styles(mp-toggle)
```

without the +? tried, didn't work. stylus is so terrible with silent failures